src/Language/Fixpoint/Misc.hs

```diff
@@ -425,12 +425,15 @@ allCombinations xs = assert (all ((length xs == ) . length)) $ go xs
 powerset :: [a] -> [[a]]
 powerset xs = filterM (const [False, True]) xs
 
+infixl 9 =>>
 (=>>) :: Monad m => m b -> (b -> m a) -> m b
 (=>>) m f = m >>= (\x -> f x >> return x)
 
+infixl 9 <<=
 (<<=) :: Monad m => (b -> m a) -> m b -> m b
 (<<=) = flip (=>>)
```

Comment:
I would think these should be removed from here. And replaced in the code with `>>=` and `=<<`.

Reply:
Looks like (=>>) :: Monad m => m b -> (b -> m a) -> m b and (<<=) are not used.

Reply:
Had a quick peek at `liquidhaskell`. Each operator is only used once there, `(=>>)` in `Language.Haskell.Liquid.Constraint.Fresh` and `(<<=)` in `Language.Haskell.Liquid.Constraint.Generate`. Perhaps those modules should define such an operator privately or use the inline equivalent?